### PR TITLE
CronJob & dev updates

### PR DESCRIPTION
**Pull request by yuito2k**
Sorry man! I accidentally deleted the previous PR to delete a single message sent by mistake. My bad! I am new to these stuffs. Well anyways, as you have told me earlier to set a CronTrigger to refresh the caches and also to update the requirements-dev.txt.
All is done but I am not sure about the requirements-dev.txt Did I do something wrong while updating the dev file?

### CODE REVIEW

**Comment by Aoi-Yuito** [comment]
Alright I understood what you meant by to update the requirements-dev.txt.

**Comment by parafoxia** [comment]
For the requirements (and more generally the checks as a whole), if you do `pip install nox` then run `nox`, you'll run the checks. Running `nox -s dependencies` will tell you which requirements need to be updated in the files, and to what versions.

I would reset the requirements file to what was there before, then run the nox check, and make the changes it recommends.

I'll be able to check the rest of it when I'm on my computer. Though those nox checks should inform you how to fix your formatting and import errors.

**Comment by Aoi-Yuito** [comment]
No worries! I fixed the dependency error but I am clueless about the import errors.

**Comment by parafoxia** [comment]
Looks like you fixed them lmao. One of the times it fails is if they're not formatted correctly, so fixing the formatting can sometimes fix the imports too, which is what's happened here. Changes you've made in the last commit look good.

File: carberretta/extensions/rtfm.py [hunk 1]
```diff
@@ -69,6 +86,10 @@ async def on_started(_: hikari.StartedEvent) -> None:
     py = await plugin.bot.d.session.get(PYTHON_DOCS_URL + "objects.inv")
     plugin.bot.d.python_cache = decode_object_inv(await py.read())
```
Comment (parafoxia) [inline carberretta/extensions/rtfm.py:87]: These two lines can be removed.

File: carberretta/extensions/rtfm.py [hunk 2]
```diff
@@ -150,6 +171,25 @@ async def python_rtfm(ctx: lightbulb.SlashContext) -> None:
...
+
+    lb = await plugin.bot.d.session.get(LIGHTBULB_DOCS_URL + "objects.inv")
+    plugin.bot.d.lightbulb_cache = decode_object_inv(await lb.read())
+
+    py = await plugin.bot.d.session.get(PYTHON_DOCS_URL + "objects.inv")
+    plugin.bot.d.python_cache = decode_object_inv(await py.read())
+
+    await ctx.respond("All RTFM caches has been refreshed.")
+
+
```
Comment (parafoxia) [inline carberretta/extensions/rtfm.py:192]: This command can be removed now it's done automatically.

**Review by parafoxia — CHANGES_REQUESTED** [review]

File: carberretta/extensions/rtfm.py (hunk 2 above)
Comment (Aoi-Yuito) [reply carberretta/extensions/rtfm.py:192]: ahh...well, I kept the command for emergency cases for example -> right after triggering this CronJob, the libs got updated. Then we will get the latest cache 6 hours later. Wouldn't it be a problem? That's why I kept it but should I still remove the command?

Comment (parafoxia) [reply carberretta/extensions/rtfm.py:192]: If you want it every 6 hours, it needs to be `hour="0,6,12,18"` -- it's currently set to only run at 6am every day. But yeah we it doesn't matter if it's a few hours late to update.

Comment (Aoi-Yuito) [reply carberretta/extensions/rtfm.py]: Alright, done!

File: carberretta/extensions/rtfm.py [hunk 3]
```diff
@@ -69,6 +72,21 @@ async def on_started(_: hikari.StartedEvent) -> None:
...
+    log.info(f"Refreshed all RTFM caches.")
+
+
+@plugin.listener(hikari.StartedEvent)
+async def on_started(_: hikari.StartedEvent) -> None:
+    hk = await plugin.bot.d.session.get(HIKARI_DOCS_URL + "objects.inv")
+    plugin.bot.d.hikari_cache = decode_object_inv(await hk.read())
+
+    lb = await plugin.bot.d.session.get(LIGHTBULB_DOCS_URL + "objects.inv")
+    plugin.bot.d.lightbulb_cache = decode_object_inv(await lb.read())
```
Comment (parafoxia) [inline carberretta/extensions/rtfm.py:84]: I think I did say these lines can be removed didn't I? If you want to call it on boot, which might be a good idea tbh, just call the function.

**Review by parafoxia — CHANGES_REQUESTED** [review]

File: carberretta/extensions/rtfm.py (hunk 3 above)
Comment (Aoi-Yuito) [reply carberretta/extensions/rtfm.py:84]: Ohh, sorry, I misunderstand. Working on it...

Comment (Aoi-Yuito) [reply carberretta/extensions/rtfm.py]: Done!

**Review by parafoxia — APPROVED** [review]

**Pull request merged**